chains/orchestrator-relays/runtime/dancelight/src/tests/common/xcm/mocks.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+use snowbridge_beacon_primitives::{
+    types::deneb, BeaconHeader, ExecutionProof, VersionedExecutionPayloadHeader,
+};
+
+pub fn mock_execution_proof() -> ExecutionProof {
```

Comment:
we have this duplicated if I am not mistaken (although in another place it's called dummy_prood. I would put it in test/common/mod.rs and re-use it in both places